mail_message_name_search/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Mail message name search
+========================
+
+This module adds the capability to search for mail messages by subject or
+body of the message. This will be useful in models that make intense use of
+messages, like project issues or helpdesk tickets.
+
+This module will add dynamically the message_ids to the search view of 
+any model that inherits from the mail.thread and will incorporate the
+capability to search for content in the mail messages.
+
+
+Installation
+============
+
+This module depends on the module 'base_search_fuzzy' to ensure that
+searches on emails are based on indexes. Please read carefully the install
+instructions https://github.com/OCA/server-tools/blob/8.0/base_search_fuzzy/README.rst
```

Comment:
Change to https://github.com/OCA/server-tools/blob/9.0/base_search_fuzzy/README.rst